src/main/resources/application.properties

```diff
@@ -19,3 +19,4 @@ quarkus.kubernetes.service-type=load-balancer
 
 quarkus.http.enable-compression=true
 quarkus.http.enable-decompression=true
+quarkus.qute.strict-rendering=true
```

Comment:
FYI this is not necessary - it's the default...

Reply:
Ah, nice